slang.h

```diff
@@ -4423,15 +4443,20 @@ namespace slang
             IBlob**     outCode,
             IBlob**     outDiagnostics = nullptr) = 0;
 
-            /** Get the name for the specified shader file containing code for the entry point at
-                'entryPointIndex' for the chosen 'targetIndex'
+            /** Compute the hash code of all compilation arguments for the specified target. This includes
+                all dependent source file names, preprocessor defines, target options and other compiler options.
```

Comment:
The comment isn't accurate, for the linkage, we no longer compute hash for preprocessor defines and other compiler options here.

Reply:
Fixed in new push